src/Symfony/Component/HttpKernel/Attribute/ControllerAttributeInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Symfony\Component\HttpKernel\Attribute;
+
+/**
+ * Marker interface for controller attributes
```

Comment:
For controller argument attributes we decided to [remove the marker interface](https://github.com/symfony/symfony/pull/40307), should we do the same here ?

Reply:
Would you suggest a different kind of marker or none at all?

Reply:
No marker at all, the same way controller argument metadata works. We might need to introduce a `ControllerMetadata` class to mirror the `ArgumentMetadata` one.